packages/kbn-alerts-ui-shared/src/maintenance_window_callout/index.tsx

```diff
@@ -76,7 +125,16 @@ export function MaintenanceWindowCallout({
 
   return (
     <EuiCallOut
-      title={MAINTENANCE_WINDOW_RUNNING}
+      title={
+        !categoryNames
+          ? MAINTENANCE_WINDOW_NO_CATEGORY_TITLE
+          : i18n.translate('alertsUIShared.maintenanceWindowCallout.maintenanceWindowActive', {
+              defaultMessage: 'Maintenance window is running for {categories} rules',
```

Comment:
>              defaultMessage: 'Maintenance window is running for {categories} rules',

This message ("Maintenance window is...") seems to appear even when there are multiple maintenance windows running (e.g. you have two different maintenance windows running concurrently for different categories).  Is it possible to have a different message when multiple windows are runnning?  If not, IMO something like this is more accurate:

> One or more maintenance windows are running for {categories} rules

Reply:
How about `{activeWindowCount, plural, one {Maintenance window is} other {Maintenance windows are}}`?

Reply:
LGTM, thanks!